crates/zombie_actor/src/ghost_channel.rs

```diff
@@ -58,6 +61,7 @@ impl<
     > GhostMessage<RequestToSelf, RequestToOther, RequestToSelfResponse, Error>
 {
     fn new(
+        requester_bt: backtrace::Backtrace,
```

Comment:
This is interesting- I like the idea of keeping the backtraces around but wonder if this more appropriate to push into the tracing layer?